samples/msal-node-samples/client-credentials-distributed-cache/src/RedisClientWrapper.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ICacheClient } from "@azure/msal-node";
+import { RedisClientType } from "redis";
+
+/**
+ * Simple persistence client helper, using Redis (node-redis). You must have redis installed
+ * on your machine and have redis server listening. Note that this is only for illustration,
+ * and you'll likely need to consider cache eviction policies and handle cache server connection
```

Comment:
Please add 2 simple cache eviction policies to showcase how this is done: 

1. a max size or max number of items
2. a default TTL set to 24h

Reply:
Gotcha -will add this and carry the same code to other distributed samples

Reply:
done and done